### DISCUSSION TITLE
Companies / Sites using Stitches

### DISCUSSION THREAD

**Discussion by peduarte** [stitchesjs/stitches#768] [Show and tell]
Are you using Stitches for your site, or at your company? Share here!

To kick us off:
- CodeSandbox — https://codesandbox.io
- Vercel – https://vercel.com
- Foundation – https://foundation.app

🚀

Using Radix too? [Share here](https://github.com/radix-ui/primitives/discussions/850)!

**Comment by cometkim** [comment]
✋ Karrot
- https://ca.karrotmarket.com
- https://uk.karrotmarket.com
- https://us.karrotmarket.com
- https://jp.karrotmarket.com
- https://team.daangn.com

Some of the microfrontend in the product and many of [websites](https://github.com/daangn/websites) are using stitches! And we will build own design system with stitches as the css runtime 😁

**Reply by peduarte** [reply]
Love them <3

**Comment by murillo94** [comment]
Magnetis — https://magnetis.com.br

We are using stitches in almost all the public pages :D

**Reply by peduarte** [reply]
Muito bom!

**Comment by msanroman** [comment]
I'm using it for anything and everything!

- Otter – https://otter.es (we are using it for the web app, and will be launching a new home soon fully styled with Stitches too!). We will most likely wrap the year with a new design system fully powered with Stitches 🚀 .
- Maria Carvajal – https://mariacarvajal.es
- Mike San Román (👋 🤓 ) – https://msanroman.io

**Reply by peduarte** [reply]
Love all three of them!

**Comment by stuarthendren** [comment]
https://committed.io/components
https://github.com/commitd/components

We are using stitches and radix for our design system that's used in many projects.

**Reply by peduarte** [reply]
Awesome! Wanna add it to the [Radix Discussion](https://github.com/radix-ui/primitives/discussions/850) too?

**Comment by v3dillon** [comment]
https://1upblitz.com uses stitches for the landing page, and radix for included components.

**Comment by hwhmeikle** [comment]
https://www.alphero.com/ - our site isn't using Stitches (I wish) but we've started using it on new projects for our clients.

We're currently making a very important website for the NZ Government that allows users to see their COVID-19 vax/test records etc. It's currently in a closed pilot, but I will definitely share a link when it has it's GA release (hopefully within the next month).

**Comment by mathisobadia** [comment]
https://mysho.ai/ it's currently a very minimal beta app, but it's going to grow into an app for local newspapers. Each newspaper will get it's own custom stitches theme. You can already see on the app the first controls we built, you can change your typography colors etc... and it will update the currently used stitches theme config values. We are building a design system with stitches and radix-ui! We couldn't be more happy with it especially since stitches went into v1 thank you @peduarte !

**Reply by MichaelDM** [reply]
Thank you @peduarte , we're embracing Stitches fully and loving it!

**Reply by peduarte** [reply]
That's awesome! 💯

**Comment by tarngerine** [comment]
just launched today! https://sprout.place/

read more: https://www.producthunt.com/posts/sprout-1-0

**Comment by franklinjavier** [comment]
@grupoboticario 
flora.ds = stitches+radix

**Comment by bryanveloso** [comment]
https://altair.tv (as well as our design system) uses Stitches and Radix. Our app isn't live yet, but it will be powered by both when it does launch.

**Comment by Perni1984** [comment]
https://www.pcode.at and our internal design system is using Stitches. Many thanks to the whole team of Modulz - you did a really awesome job guys.

**Comment by frgarciames** [comment]
We built our UI library with stitches and radix
https://github.com/finetwork-os/ui
https://finetwork-ui.vercel.app/
Also in the private area of our users
https://mifi.finetwork.com/
We're using in one project that we're developing, and of course in every new project that we develop !

**Comment by nixjs** [comment]
I built my website with stitches and tailwind. 
https://casino-gambling-nixjs.vercel.app/dice
